<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="29118" started-at="2020-01-03T08:55:38Z" finished-at="2020-01-03T08:56:07Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="29118" started-at="2020-01-03T08:55:38Z" finished-at="2020-01-03T08:56:07Z">
      <class name="firstTest.HomePageTest">
        <test-method status="PASS" signature="navigatepages()[pri:0, instance:firstTest.HomePageTest@1dab4eb]" name="navigatepages" duration-ms="9851" started-at="2020-01-03T14:25:38Z" finished-at="2020-01-03T14:25:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigatepages -->
        <test-method status="FAIL" signature="subNewsLetter()[pri:0, instance:firstTest.HomePageTest@1dab4eb]" name="subNewsLetter" duration-ms="18784" started-at="2020-01-03T14:25:48Z" finished-at="2020-01-03T14:26:06Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Not Displayed expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Not Displayed expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at firstTest.HomePageTest.subNewsLetter(HomePageTest.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- subNewsLetter -->
        <test-method status="PASS" signature="veriftabs()[pri:0, instance:firstTest.HomePageTest@1dab4eb]" name="veriftabs" duration-ms="380" started-at="2020-01-03T14:26:06Z" finished-at="2020-01-03T14:26:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- veriftabs -->
      </class> <!-- firstTest.HomePageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
